computeResource.go

```diff
@@ -6,6 +6,17 @@ import (
 	"net/http"
 )
 
+// VirtualizationTypeName represents available storage types.
```

Comment:
Not "storage types" but virtualization types.

Reply:
done